Captstone project

Wikitelmex opened the pull request:
- linters fixed
- desk version
- responsive mobile first aproach
- github deployment
- live demo
- credits to the owner of the template
- add descriptive video
- add add readme sections getstarted, notes, explaining video
- transpiled file (style.css) avoided to being ignored to have a live demo working (and avoided the linter errors that this transpiled file generates)
- Style.SCSS without linter errors (non transpiled file)

wikitelmex1 committed:
linters fixed

wikitelmex1 committed:
stylishing improvements

wikitelmex1 committed:
style linter error fixed

wikitelmex1 committed:
readme file

- add readme.md
- add screenshot

wikitelmex1 committed:
Speakers JavaScript

- Speakers Loaded Dinamically
- Deleted Static Cards

wikitelmex1 committed:
Update script.js

- linters fixed

Georjane commented on README.md:
```diff
@@ -1,2 +1,41 @@
+[Live Demo Link](https://wikitelmex.github.io/capstone-module1/)
+
+
+
```
- [x] Please can you kindly add a link to your video presentation to your Readme file. You were required to do a video presentation of your project
- [x] Also please can you add instructions on how to clone your project locally and get it running. For example
```
## Getting Started
To get a local copy of the repository please run the following commands on your terminal:
$ git clone https://github.com/....
$ cd ...
$ git checkout ...
$ Open index.html file in your browser or with live server
```

Georjane commented on README.md:
```diff
@@ -1,2 +1,41 @@
+
+## Live Demo
+
+[Live Demo Link](https://wikitelmex.github.io/capstone-module1/)
```
- [x] Can you please check your deployed code? This is what I have at my end which doesn't display like the screenshot in your readme file.
![image](https://user-images.githubusercontent.com/60772732/132940404-54d3cb90-984c-4e06-95ab-dca8155381f3.png)

Georjane commented on scss/style.scss:
```diff
@@ -54,6 +71,17 @@ $spacers: (
 
+$btn-border-radius: 0;
+
+@import "../node_modules/bootstrap/scss/bootstrap";
```
- [ ] Please can you add instructions for compiling `scss `to `css `in your readme file? I tried that using ` sass --update scss:css` but it throws the following error that it can't find this module. Since your node modules were not pushed to GitHub, you can create a package.json file to resolve this so I can generate the node modules on my end too when I run a command like `npm install` for example
```
$ sass --update scss:css
      error scss/style.scss (Line 76: File to import not found or unreadable: ../node_modules/bootstrap/scss/bootstrap.
Load path: C:/Users/PC/Desktop/TSE/capstone-module1/scss)

```

Georjane commented on index.html:
```diff
@@ -122,43 +144,54 @@ <h2 class="text-primary  col-4">Prizing</h2>
+        <footer class="bg-light text-dark">
+            <div class="footer-info">
+                <div class="me-2 footer-logo">
+                    <img class="me-1 logo" src="./assets/Logo.svg" alt="">
```
- [ ] Please can you add an alt attribute value to this image. This will help screen readers to identify images. Please do the same for all other img elements in both the inde.html file and the about.html file too

Georjane reviewed (changes requested):
### Hello, @Wikitelmex, ✋🏽 

Good job so far! Your PR is professional and you have a good mastery of git-flow. Your linters are all passing, I applaud that too.
However, we have a few more changes to make in order to meet all requirements for this project but you are almost there!

## Required Changes ♻️

- [x] I was unable to check your Page Structure and your Design and styling because the live version seems to have a problem and also you did not provide instructions on how to compile scss to css in your readme file. 

_Check the comments under the review for other required changes._

Cheers and Happy coding! 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 🤝 

_Code Reviewer
**[Jane](https://witahgeorjane.tech/)**_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

wikitelmex1 committed:
Update index.html

wikitelmex1 committed:
fixing deployment

wikitelmex1 committed:
deployment fixing

wikitelmex1 committed:
project improvements

- add descriptive video
- add add readme sections getstarted, notes, explaining video
- transpiled file (style.css) avoided to being ignored to have a live demo working (and avoided the linter errors that this transpiled file generates)
- Style.SCSS without linter errors (non transpiled file)

wikitelmex1 committed:
change video link

- readme.md doesn't allow embedded videos, change to single link

wikitelmex1 committed:
font cocogoose issue

wikitelmex1 committed:
cocogoose font fix

wikitelmex1 committed:
font issues

wikitelmex1 committed:
font fixing

wikitelmex1 committed:
Update style.css

wikitelmex1 committed:
update style

wikitelmex1 committed:
improvements

vichuge commented on about.html:
```diff
@@ -48,27 +53,28 @@
-                <h1 class="text-primary fw-bold fs-1 pb-3">SOFTWARE DEVELOPING CONTEST 2021</h1>
+            <h2 class="text-primary mt-5 pb-3 mobile-visible">"Hello! Developers"</h2>
+            <div class="tw-400 text-container-center">
+                <h1 class="text-primary fw-bold fs-1 pb-3 text-element-center">SOFTWARE DEVELOPING CONTEST 2021</h1>
```
- [ ] It looks like you have a bug with the title on mobile size, please fix it. 🐶 
![image](https://user-images.githubusercontent.com/20563252/132961377-d8979dd4-f845-4d89-afd2-c1302d7cef43.png)

vichuge commented on about.html:
```diff
@@ -48,27 +53,28 @@
+                <h1 class="text-primary fw-bold fs-1 pb-3 text-element-center">SOFTWARE DEVELOPING CONTEST 2021</h1>
+                <div class="card bg-secondary border-1 mw-60vh">
+                    <div class="card-body">
+                        A friendly competition celebrated in Mexico, participating countries all over the world, this december 2021
```
- [x] The mobile bug is here too. Please fix it 🐱 
![image](https://user-images.githubusercontent.com/20563252/132961395-b4a1145c-5347-4b9d-a661-3ff666323ff6.png)

vichuge commented on about.html:
```diff
@@ -81,57 +87,63 @@ <h2>South Korea</h2>
-                  <h3 class="card-title">2019</h3>
-                  <h6 class="card-subtitle mb-2">Winner: Japan</h6>
-                  <p class="card-text">Location: Brazil</p>
+            <div class="presenters">
+                <div class="card text-center hero-image text-light mb-2 tw-400">
+                    <div class="card-body hero-div">
+                      <h3 class="card-title mt-2">2019</h3>
+                      <h6 class="card-subtitle mb-2">Winner: Japan</h6>
+                      <p class="card-text mb-2">Location: Brazil</p>
+                    </div>
```
- [x] The size bug is here too. Please, work on it. 🌵 
![image](https://user-images.githubusercontent.com/20563252/132961413-f2cb59a3-7d77-4db8-b500-07bc57cfd962.png)

vichuge commented on about.html:
```diff
@@ -22,6 +26,7 @@
+                    <button class="navbar-closer mobile-visible" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent">&#10005;</button>
                     <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
                         <li class="nav-item">
                             <a class="nav-link active" aria-current="page" href="./about.html">About</a>
```
- [ ] For the navbar please, add the black one on the top. Right now you only have the white.
Your project:
![image](https://user-images.githubusercontent.com/20563252/132961444-5e52548e-b503-44cf-b6e3-a396230d779f.png)

The template:
![image](https://user-images.githubusercontent.com/20563252/132961461-61edbcc8-eb58-4704-8fd1-eaef71aec8a9.png)

vichuge commented on about.html:
```diff
@@ -81,57 +87,63 @@ <h2>South Korea</h2>
-            <div class="d-flex flex-row p-1">
-                <div class="me-2">
+            <div class="footer-info">
+                <div class="me-2 footer-logo">
                     <img class="me-1 logo" src="./assets/Logo.svg" alt="">
                 </div>
                 <div>
-                    <p><small>2021 Alex Dev Mexico.</small></p>
-                    <p><small>All Rights Reserved</small></p>
+                    <p><small>2021 Alex Dev Mexico. All Rights Reserved</small></p>
+                    <p class="mobile-hidden"><small>Alex Dev company, is a non-profit company, for any purpose related to this, please contact wikitelmex@gmail.com</small></p>
                 </div>
             </div>
+            br
         </footer>
```
- [ ] Please, consider make this for the footer: 🦊 
![image](https://user-images.githubusercontent.com/20563252/132961497-beeba74f-1cc9-43c6-8324-acd2a47457a8.png)

vichuge commented on index.html:
```diff
@@ -122,43 +146,54 @@ <h2 class="text-primary  col-4">Prizing</h2>
         <div class="bg-light text-dark p-2">
             <br>
             <h2 class="text-center">Featured Speakers</h2>
```
- [ ] Talking about "featured speakers" part. Is important to add the 6 elements. At same time, please, add the arrow to show more when you're on mobile size.
![image](https://user-images.githubusercontent.com/20563252/132961546-ebe8d617-54df-40ca-a501-d14612c4ec61.png)
![image](https://user-images.githubusercontent.com/20563252/132961554-e79ce013-29bc-430b-beb4-4e025d34583b.png)

- Hint: I understand the template could show the home page on mobile size doesn't have a footer, but please, don't forget to add it.

vichuge commented on about.html:
```diff
@@ -81,57 +87,63 @@ <h2>South Korea</h2>
+                <div class="card text-center hero-image text-light mb-2 tw-400">
+                    <div class="card-body hero-div">
+                      <h3 class="card-title mt-2">2019</h3>
+                      <h6 class="card-subtitle mb-2">Winner: Japan</h6>
```
- [ ] I'm sure on the images, you could add a space between them (on desktop size) and have it more centered. Please work on this. 🐔 
![image](https://user-images.githubusercontent.com/20563252/132961619-182f63f2-18f3-4001-be7f-5890c59b395c.png)

vichuge commented on about.html:
```diff
@@ -81,57 +87,63 @@ <h2>South Korea</h2>
-                <div class="me-2">
+            <div class="footer-info">
+                <div class="me-2 footer-logo">
                     <img class="me-1 logo" src="./assets/Logo.svg" alt="">
```
- [ ] Please add an alt value for every image on your project, this helps for SEO purposes and has a better qualification for google. 🪐

vichuge commented on index.html:
```diff
@@ -69,34 +73,54 @@ <h2 class="fw-bold">2021.12.17(FRI) - 19(SUN)</h2>
                 
-                <div class="container-sm bg-dark-hopacity row mb-1 ms-1 pt-2 pb-2 tw-400">
+                <div class="container-sm bg-dark-hopacity row mb-1 ms-1 pt-2 pb-2 tw-400-card">
+                    <img class="col-2 img-cards" src="./assets/work.svg" alt="">
```
- [ ] Please add an alt value for every image on your project, this helps for SEO purposes and has a better qualification for google. 🪐

vichuge commented on index.html:
```diff
@@ -69,34 +73,54 @@ <h2 class="fw-bold">2021.12.17(FRI) - 19(SUN)</h2>
+                </div>
+
+                <div class="container-sm bg-dark-hopacity row mb-1 ms-1 pt-2 pb-2 tw-400-card">
+                    <img class="col-2 img-cards" src="./assets/work.svg" alt="">
```
- [ ] Please add an alt value for every image on your project, this helps for SEO purposes and has a better qualification for google. 🪐

vichuge commented on index.html:
```diff
@@ -69,34 +73,54 @@ <h2 class="fw-bold">2021.12.17(FRI) - 19(SUN)</h2>
+                </div>
+
+                <div class="container-sm bg-dark-hopacity row mb-1 ms-1 pt-2 pb-2 tw-400-card">
                     <img class="col-2 img-cards" src="./assets/work.svg" alt="">
```
- [ ] Please add an alt value for every image on your project, this helps for SEO purposes and has a better qualification for google. 🪐

vichuge commented on index.html:
```diff
@@ -69,34 +73,54 @@ <h2 class="fw-bold">2021.12.17(FRI) - 19(SUN)</h2>
 
-                <div class="container-sm bg-dark-hopacity row mb-1 ms-1 pt-2 pb-2 tw-400">
+                <div class="container-sm bg-dark-hopacity row mb-1 ms-1 pt-2 pb-2 tw-400-card">
                     <img class="col-2 img-cards" src="./assets/work.svg" alt="">
```
- [ ] Please add an alt value for every image on your project, this helps for SEO purposes and has a better qualification for google. 🪐

vichuge commented on index.html:
```diff
@@ -69,34 +73,54 @@ <h2 class="fw-bold">2021.12.17(FRI) - 19(SUN)</h2>
 
-                <div class="container-sm bg-dark-hopacity row mb-1 ms-1 pt-2 pb-2 tw-400">
+                <div class="container-sm bg-dark-hopacity row mb-1 ms-1 pt-2 pb-2 tw-400-card">
                     <img class="col-2 img-cards" src="./assets/work.svg" alt="">
```
- [ ] Please add an alt value for every image on your project, this helps for SEO purposes and has a better qualification for google. 🪐

vichuge commented on index.html:
```diff
@@ -122,43 +146,54 @@ <h2 class="text-primary  col-4">Prizing</h2>
+        <footer class="bg-light text-dark">
+            <div class="footer-info">
+                <div class="me-2 footer-logo">
+                    <img class="me-1 logo" src="./assets/Logo.svg" alt="">
```
- [ ] Please add an alt value for every image on your project, this helps for SEO purposes and has a better qualification for google. 🪐

vichuge reviewed (changes requested):
# Required Changes ♻️ ♻️ ⚠️ ⚠️ 🚦 🚦 

Hi dear student, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but some are still not addressed yet.

![star_wars_almost_there](https://user-images.githubusercontent.com/20563252/121369354-644f4900-c901-11eb-9844-50c85d0f206b.gif)


## Aspects to improve ♻️
_Check the comments under the review._

- [ ] Check the comments under the review. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

wikitelmex1 committed:
second review changes required

- add navbar dark with social media
- bugs fixed
- stylishing added

mricanho reviewed (approved):
# Status: ✔️ Project Approved ✔️ 

Hello @Wikitelmex  🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
